job-service-db-client/pom.xml

```diff
@@ -0,0 +1,50 @@
+<!--
+
+    Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.github.jobservice</groupId>
+    <artifactId>job-service-db-client</artifactId>
+    <name>job-service-db-client</name>
+
+    <parent>
+        <groupId>com.github.jobservice</groupId>
+        <artifactId>job-service-aggregator</artifactId>
+        <version>2.4.0-SNAPSHOT</version>
+    </parent>
+
+    <build>
+        <finalName>job-service-db-client</finalName>
+        <sourceDirectory>src/main/java</sourceDirectory>
+    </build>
```

Comment:
Why is build node present?

Reply:
Unrequired, I have removed this in the latest commit.